be/src/exec/mysql_scanner.cpp

```diff
@@ -117,7 +117,7 @@ Status MysqlScanner::query(const std::string& query) {
 Status MysqlScanner::query(const std::string& table, const std::vector<std::string>& fields,
                            const std::vector<std::string>& filters,
                            const std::unordered_map<std::string, std::vector<std::string>>& filters_in,
-                           std::unordered_map<std::string, bool>& filters_null_in_set, int64_t limit) {
+                           std::unordered_map<std::string, bool>& filters_null_in_set, int64_t limit, std::string& temporal_clause) {
```

Comment:
Please consider make it `const std::string&` .

Reply:
Added in latest commit